lib/common.js

```diff
@@ -1,35 +1,28 @@
 'use strict';
 
-const common = {};
-module.exports = common;
-
 // Forward an event from one EventEmitter to another.
 //   from - EventEmitter to listen for event
 //   to - EventEmitter to emit event on
 //   eventName - name of the event
 //   newEventName - name of the forwarded event (optional)
 //
-common.forwardEvent = (from, to, eventName, newEventName = eventName) => {
-  from.on(eventName, (...eventArgs) => {
-    to.emit(newEventName, ...eventArgs);
-  });
+const forwardEvent = (from, to, eventName, newEventName = eventName) => {
+  from.on(eventName, (...eventArgs) => to.emit(newEventName, ...eventArgs));
```

Comment:
IMO using the lambda expression syntax emphasizes the significance of the return value. Besides that, I find the prior version easier to read because of shorter lines.